source/renderer/app/components/widgets/splash/Splash.js

```diff
@@ -18,6 +18,7 @@ type Props = {
   description: Node,
   buttonLabel: string,
   linkLabel: string,
+  isIncentivizedTestnetTheme?: boolean,
```

Comment:
@DeeJayElly please make this one mandatory prop.

Reply:
It's optional for another Splash screen, <SplashNetworkFlight> doesn't need to pass it to <Splash> component.

Reply:
> It's optional for another Splash screen - <SplashNetworkFlight, doesn't need to pass it to component.

Reply:
Yes so make it mandatory on the parent as well but not on the other splash screen.

Reply:
Done